packages/cli/config/eslint-factory.js

```diff
@@ -66,7 +66,7 @@ function createConfig(dir, extraConfig = {}) {
       ...(extraExtends ?? []),
     ],
     parser: '@typescript-eslint/parser',
-    plugins: ['import', 'unused-imports', ...(plugins ?? [])],
+    plugins: ['import', 'unused-imports', 'deprecation', ...(plugins ?? [])],
```

Comment:
This should not be required to be done explicitly as the plugin is part of `@typescript-eslint`, right? ([Source](https://github.com/gund/eslint-plugin-deprecation?tab=readme-ov-file#prerequisites))

Reply:
That seems to be specific to using `@typescript-eslint/plugin`, though? Which we don't, AFAICT.

Reply:
... also, this is just about the parser setup, the way that I read the docs over there. Not about this plugin actually being included in the top-level plugin.